### DISCUSSION TITLE
blog/list-animation

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#425] [General]
# blog/list-animation

An in-depth guide on how to animate enter and exit animation for list using Motion for React (previously Framer Motion).

https://web-v2-git-new-motion-blog-theodorus-clarences-projects.vercel.app/blog/list-animation

**Comment by theodorusclarence** [comment]
Hello, kindly leave a like and comment if you got something to ask! 🚀
